### ISSUE TITLE
Depreciation warnings for dependencies during installation

### ISSUE THREAD

**Issue by ahmedelq** [mapbox/node-pre-gyp#877]
When installing `@mapbox/node-pre-gyp` I get deprecation warnings from npm issues for the following packages:

- `npmlog@5.0.1`
- `are-we-there-yet@2.0.0`
- `gauge@3.0.2`

**Environment**:
- npm version: 10.2.4
- node version: 21.6.2
- OS: Ubuntu 22.04

**Related PRs**:
* [Replace `npmlog` with `consola` mapbox/node-pre-gyp#829](https://github.com/mapbox/node-pre-gyp/pull/829)

**Comment by cclauss** [comment]
https://github.com/mapbox/node-pre-gyp/pulls/app%2Fdependabot Pull requests will resolve these over time.

**Comment by FAbrahamDev** [comment]
It seems like its been a minute since these pull requests have been worked on.
And i'd like to add `rimraf@3.0.2` that produces this warning:
```
 WARN  deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
```

As this module is used by pdfjs, with millions of downloads. I cant be the only one with memory leaks now in their code now :D
```
@tato30/vue-pdf 1.11.0
└─┬ pdfjs-dist 4.5.136
  └─┬ canvas 2.11.2
    └─┬ @mapbox/node-pre-gyp 1.0.11
      └─┬ rimraf 3.0.2
        └─┬ glob 7.2.3
          └── inflight 1.0.6
```

**Comment by cclauss** [comment]
What is the status of this?

**Comment by benmccann** [comment]
I believe this should be fixed in 2.0.0-rc.0
